### DISCUSSION TITLE
dark theme?

### DISCUSSION THREAD

**Discussion by lephtHanded** [wg-easy/wg-easy#258] [General]
are there any plans to create a dark theme for the UI?

**Comment by eandersons** [comment]
Currently there are two open pull requests related to dark theme: #178 and #250.

**Comment by pheiduck** [comment]
It is still on the road: https://github.com/wg-easy/wg-easy/pull/691
Until now it is in testing state: `ghcr.io/wg-easy/wg-easy:nightly`
Report issues here: https://github.com/wg-easy/wg-easy/issues
